src/Components/Browser.JS/Microsoft.AspNetCore.Components.Browser.JS.npmproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
 
   <PropertyGroup>
-    <IsTestProject>false</IsTestProject>
+    <IsTestProject>true</IsTestProject>
```

Comment:
??????? I'm not sure this is all changing.

Reply:
because it's it's own test project?

Reply:
Yeah. It's kind of how SignalR structures it's projects, so you use one `npmproj` to track building and testing stuff. Also helps that you only do one `yarn install` for srces and tests.